examples/cephfs/pvc-rwop.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: csi-cephfs-pvc
```

Comment:
better use a different name here too

Reply:
Done

Reply:
@nixpanic one thing I noticed here was, if I change the name in the yaml, I have to introduce 3 other yamls ( POD yamls) for new these PVCs. if really required I can add that, otherwise we can stick to the default POD yamls with these name. is that fine ?

Reply:
yes, that is really required. Use a name for the PVC that describes the `rwop` part. It is very confusing when users create the PVCs manually from these examples. Examples need to have descriptive names, otherwise we would use `pvc` and `pod` and `storageclass` for everything else too.